code/modules/mob/living/carbon/human/say.dm

```diff
@@ -257,27 +106,27 @@
 		verb = speaking.get_spoken_verb(ending)
 	else
 		if(ending == "!")
-			verb = "exclaims"
-		else if(ending == "?")
-			verb = "asks"
+			return pick("exclaims", "shouts", "yells")

```

Comment:
I'd leave this as just "exclaims" personally, but that's about the only thing I really noticed about this PR, for now.